consul-topology/sprawl/boot.go

```diff
@@ -121,10 +121,8 @@ func (s *Sprawl) launchType(firstTime bool) (launchErr error) {
 		}
 	}
 
-	for _, peering := range s.topology.Peerings {
-		if err := s.waitForPeeringEstablishment(peering); err != nil {
-			return fmt.Errorf("waitForPeeringEstablishment[%s]: %w", peering.String(), err)
-		}
```

Comment:
Fixing a lint regarding the `peering` arg not being used, I guess `waitForPeeringEstablishment` waits for all peerings? so I removed the loop. not sure if this is true.